doc/user-guide/github.md

```diff
@@ -125,3 +125,42 @@ If workflow needs an update a relevant message will be displayed:
 
 Please review the workflow file and update if needed.
 
+## Secrets
+
+We generally advise users to define their secrets in an `.env` file. However, this isn't optimal in the case of Github deployment, since uploading an `.env` file would expose your plain secrets.
+
+As a result, we've created a workflow where the CLI can read secrets from your local environment. To take advantage of this, follow these steps:
+
```

Comment:
I'd remove this and just straight explain what the solution is 

also, mention that the secrets should be stored as github secrets and then passed as environment variables, you can put some snippet showing how to do it: https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#example-using-bash

Reply:
[Fixed to go straight to explanation](https://github.com/ploomber/doc/pull/212/files#diff-f41679bbab33b86f374ce79abb5adcfbf4e6c05394c594cba25012bc1f48f43eR128-R135)

Added instruction to [store github secrets with snippet](https://github.com/ploomber/doc/pull/212/files#diff-f41679bbab33b86f374ce79abb5adcfbf4e6c05394c594cba25012bc1f48f43eR152-R162)